crud/common/schema.lua

```diff
@@ -244,4 +244,15 @@ function schema.result_needs_reload(space, result)
     return result.space_schema_hash ~= get_space_schema_hash(space)
 end
 
+function schema.batching_result_needs_reload(space, results, tuples_count)
+    local storage_errs_count = 0
+    for _, result in ipairs(results) do
+        if result.space_schema_hash ~= nil and result.space_schema_hash ~= get_space_schema_hash(space) then
```

Comment:
I guess we can calculate it once.

Reply:
You mean perform `get_space_schema_hash(space)` once?

Reply:
Yes.

Reply:
Done.